src/api/apiFetch.js

```diff
@@ -38,7 +38,7 @@ export const apiCall = async (
     const response = await timeout(
       await apiFetch(auth, route, params),
       () => { throw new Error(`Request timed out @ ${route}`); },
-      shouldTimeout,
+      // shouldTimeout, # WTF ITS BOOL ;-( WE NEED NUMBER
```

Comment:
Thats strange, I commented it out now - it should be a number, boolean has no sense here.